CHANGELOG.md

```diff
@@ -133,7 +133,7 @@ As a result we have added 2.0-maintenance branch which is the path to EOL for Op
 * Instead of the old version-mixed OSClient a OSClientV2 for only Identity V2 API and OSClientV3 for only Identity V3 API are available
 * Same refactoring for the OSClientSession: introducing new OSClientSessionV2 and OSClientV3
 
-### 2.20
+### 2.1.0
```

Comment:
I am not able to find the release version you are suggesting anywhere. According to [1] and [2], latest 2.X is indeed 2.20.

What makes you to believe this is incorrect?

[1] https://github.com/ContainX/openstack4j/tags
[2] https://search.maven.org/artifact/org.pacesys/openstack4j